app/assets/stylesheets/custom.scss

```
@@ -96,3 +96,18 @@ input {
 .dropdown-menu.active {
   display: block;
 }
+
+.checkbox {
+  margin-bottom: 10px;
+  margin-top   : 10px;
```

```suggestion
  margin-top: 10px;
```

Sau cứ như này mà làm chứ mình ko căn lề kiểu kia đâu nhé

---

ok anh